components/src/organisms/LabwareDetailsWithCount/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { StyledText } from '../../atoms'
+import styles from './LabwareDetailsWithCount.module.css'
+
+type LabwareDetailsWithCountProps = {
+  title: string
+  subTitle?: string
+  label?: string
```

Comment:
if this is presumed to always be a quantity, I recommend keeping it as 
```suggestion
  quantity?: number
```
and transforming to a string within the component! that way, each caller doesn't have to do the transformation, and we prevent bad strings from being passed accidentally